kiteconnect/connect.py

```diff
@@ -263,8 +265,8 @@ def generate_session(self, request_token, api_secret):
         if "access_token" in resp:
             self.set_access_token(resp["access_token"])
 
-        if resp["login_time"] and len(resp["login_time"]) == 19:
```

Comment:
May be we can do something like this and add `login_time` to `_format_response`?

```python
        resp = self._format_response(self._post("api.token", params={
            "api_key": self.api_key,
            "api_key": self.api_key,
            "request_token": request_token,
            "request_token": request_token,
            "checksum": checksum
            "checksum": checksum
        }))
```

Reply:
Didn't get you. You mean to continue using earlier string length comparison(`len(resp["login_time"]) == 19`)? Instead of current [parse(string)](https://github.com/ranjanrak/pykiteconnect/blob/fix-tz-info/kiteconnect/connect.py#L834).

Reply:
No, just add this field name to - https://github.com/zerodha/pykiteconnect/blob/bacaafc83b38655f94c86c4abb03682ee9facc7e/kiteconnect/connect.py#L402

Reply:
Done.